ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java

```diff
@@ -18,6 +18,7 @@
     private static final By EDIT = By.cssSelector("i[title='Edit Condition: ']");
     private static final By DELETE = By.cssSelector("i[title='Delete']");
     private static final By CONDITIONS_LIST = By.cssSelector("tr.clickable-tr");
+    private static final By DELETE_CONDITION = By.cssSelector("#remove-condition-dialog > div.dialog-content > button.confirm.right");
```

Comment:
@Ndacyayisenga-droid  what do you think of `CONFIRM_DELETE_CONDITION`?

Reply:
> @Ndacyayisenga-droid what do you think of `CONFIRM_DELETE_CONDITION`?

@kdaud done